caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -676,6 +694,88 @@ def save(self,filename):
         else:
             raise Exception("Unsupported file extension")
 
+#    def save_NWB(self, filename, ID, sess_desc='CaImAn Results', exp_desc=None,
```

Comment:
Why are we adding this commented-out function? (it also seems to import a lot of stuff at runtime that if we were to uncomment it should be imported at include time instead)